common/main/ff.xml

```diff
@@ -821,4 +821,8 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 			<nostr>Alaa:A</nostr>
 		</messages>
 	</posix>
+	<personNames>
+		<nameOrderLocales order="givenFirst">und ff</nameOrderLocales>
+		<nameOrderLocales order="surnameFirst">ko vi yue zh</nameOrderLocales>
+	</personNames>
```

Comment:
Just curious… what value does this add to add this to several locales?  

There doesn't seem to be any documentation for these elements in the spec.  But does does `<nameOrderLocales order="givenFirst">und</nameOrderLocales>` already apply to all locales?  Why add this to, say, ff?

The "surnameFirst" data is also in root, is there a reason to repeat it here?

Reply:
These are in the design doc (haven't made it into LDML yet, but will). They say:

For formatting these languages in this locale, use the surnameFirst or givenFirst patterns, respectively.

Reply:
Perhaps the spec could link to the design doc as an interim step? 

My specific question was if `und ff` was any different than `und` where ff is not mentioned.